misc/testerina/modules/testerina-runtime/src/main/java/org/ballerinalang/test/runtime/entity/PartialCoverageModifiedLine.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
This year should be the year that we committed this file for the first time. Thus it should be 2021. Check and fix other files in the same PR as well.

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/28767/commits/fffea3e92edd4a82053fde14e89e1d52f4c3b4f5